projects/Mallard/src/hooks/use-fetch.ts

```diff
@@ -96,15 +97,16 @@ const useFetch = <T>(
                             })
                         }
                     })
-                    .catch(() => {
+                    .catch(err => {
+                        console.error(err)
```

Comment:
```suggestion

```

Reply:
We've got logs earlier on in this file for clearing the cache, I thought this one might be similarly useful but happy to remove it